11_notify/n0npax/cmd/lostpuppy-service/main_test.go

```diff
@@ -0,0 +1,40 @@
+package main
+
+import (
+	"errors"
+	"os"
+	"testing"
+
+	"bou.ke/monkey"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestMain(t *testing.T) {
+	os.Args = []string{"", "-p", "88888888"}
+
+	fakeExit := func(int) {
+		panic("foo-arg")
+	}
+	patch := monkey.Patch(os.Exit, fakeExit)
```

Comment:
still not a huge fan of the monkey patching - i've been getting to 100% throughout my projects without using it.
happy to file that as personal preference though.

Reply:
with some hacks, I was able to get rid of `bou.ke/monkey`. Hope it's fine now. Marking conversation as resolved.